source/documentation/ingestion/cadet-registration.md

```diff
@@ -0,0 +1,34 @@
+# Adding tables to Find MoJ Data from Create a Derived Table (CaDeT)
+
+- Find MoJ Data is integrated with the [Create a Derived Table](https://github.com/moj-analytical-services/create-a-derived-table) service (CaDeT).
+- DBT models created from CaDeT are represented by tables in Find MoJ Data.
+- To add a model created in CaDeT to Find MoJ Data, add the tag `dc_display_in_catalog` to that model. [Config of CaDeT models is described in their documentation here.](https://user-guidance.analytical-platform.service.justice.gov.uk/tools/create-a-derived-table/models/#where-can-i-define-configs)
+- To add an owner to a table in Find MoJ Data, add the `dc_owner` metadata item to the CaDeT model. This represents the data owner of the data held within the model.
```

Comment:
We can be a bit more specific about the owner information. We need users to enter owners as their name appears in their @justice.gov.uk email. e.g. `matthew.laverty123@justice.gov.uk` would be `dc_owner: matthew.laverty123`